src/tests/commonTests.ts

```diff
@@ -430,125 +430,134 @@ interface LabelableOptions extends Pick<FocusableOptions, "focusTargetSelector"
 /**
  * Helper for asserting label clicking functionality works.
  *
- * @param {string} componentTagOrHtml - the component tag or HTML used to test label support
- * @param {LabelableOptions} [options] - labelable options
+ * Note that this helper should be used within a describe block.
+ *
+ * @example
+ * describe("labelable", () => {
+ *    async () => labelable("calcite-button")
+ * })
+ *
+ * @param {string} componentTagOrHtml - The component tag or HTML used to test label support.
+ * @param {LabelableOptions} [options] - Labelable options.
  */
-export async function labelable(componentTagOrHtml: TagOrHTML, options?: LabelableOptions): Promise<void> {
-  const id = "labelable-id";
-  const labelTitle = "My Component";
-  const propertyToToggle = options?.propertyToToggle;
-  const focusTargetSelector = options?.focusTargetSelector || `#${id}`;
-  const shadowFocusTargetSelector = options?.shadowFocusTargetSelector;
-  const componentTag = getTag(componentTagOrHtml);
-  const componentHtml = isHTML(componentTagOrHtml)
-    ? ensureId(componentTagOrHtml)
-    : `<${componentTag} id="${id}"></${componentTag}>`;
-
-  function ensureId(html: string): string {
-    return html.includes("id=") ? html : html.replace(componentTag, `${componentTag} id="${id}" `);
-  }
+export function labelable(componentTagOrHtml: TagOrHTML, options?: LabelableOptions): void {
```

Comment:
Calls to `assertLabelable` should be their own test (similar to [`formAssociated`](https://github.com/Esri/calcite-components/pull/6846/files#diff-13e68f4941313174caf52190c1fe2b59180f4012d1c3e610bc72ce5ab3bbf09c)).